### ISSUE TITLE
Open each Laporan Transaksi (receipt) and re Print Button feature

### ISSUE THREAD

**Issue by astotoster** [apinprastya/sultan#41]

**Comment by idhamperdameian** [comment]
May for temporary solution you can press `F6` key in cashier tab and then print from there

**Comment by idhamperdameian** [comment]
Hi @apinprastya I get some clue may can help for development from https://stackoverflow.com/questions/2731624/how-to-get-status-from-pos-printer and https://stackoverflow.com/questions/56129924/is-there-a-way-to-get-the-status-of-a-thermal-printer-using-esc-commands.

**Comment by apinprastya** [comment]
I don't think it is necessary to implement the detection. Printing using pool does not return anything and most of cheap printer not support this. And I believe most of Sultan user using this printer.
